.github/workflows/commitlint.yml

```
@@ -5,7 +5,7 @@ jobs:
   commitlint:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4.1.7
```

Some of these actions have a major version, let's say vX, that's always updated with the latest changes: https://github.com/actions/checkout/compare/v4.1.7...v4. So we can use "v4" instead and know that's the latest version. This might be a good practice for actions where the latest versions are stable enough that we can trust its "automatic" update.

More on actions versioning: https://docs.github.com/en/actions/creating-actions/about-custom-actions#using-tags-for-release-management

```suggestion
      - uses: actions/checkout@v4
```

At least, we can do this with this action. I'm not sure about the stability of the others.

---

But if they're stable enough, I think we should use major versions.

---

Done, thanks